svo (setup, misc, empty, funnies, dor).xml

```diff
@@ -143,6 +143,8 @@ local gaffl          = svo.gaffl
 svo.gdefc            = svo.gdefc or {}
 svo.me               = svo.me or {}
 local me             = svo.me
+svo.classinfo        = {}
```

Comment:
Is there a reason this entry isn't svo.classinfo = svo.classinfo or {}? 

Would empty the table on saving the script.